library/ssl_tls.c

```diff
@@ -6916,41 +6916,6 @@ int mbedtls_ssl_read( mbedtls_ssl_context *ssl, unsigned char *buf, size_t len )
         }
     }
 
-    /*
-     * TODO
```

Comment:
Please keep this comment. It's useful even as a description of the current behavior.

Reply:
This is an almost identical change to the original PR #1446 which has already been reviewed and merged.

I don't think describing what the code should do in a maintenance branch is helpful.

Reply:
@sbutcher-arm: I personally think that it is actually more useful to describe the current behaviour and limitations in a maintenance branch than in development because we can be more aggressive with regards to what we refactor/fix/improve in the HEAD but not so much in the maintenance released because of compatibility, etc. Therefore, at some point the development could have a completely different implementation to what is here and if someone has to fix an issue with the old implementation in the maintenance release there will not be a comment describing the behaviour to help...